membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -152,6 +166,23 @@ class AttributesFromZuora(implicit val executionContext: ExecutionContext) exten
     }
   }
 
+  def getAccountSummaries(accountIds: List[AccountId],
+                       identityId: String,
+                       accountSummaryForAccountId: AccountId => Future[Disjunction[String, AccountSummary]]): Future[Disjunction[String, List[AccountSummary]]] = {
```

Comment:
better to stick to two space indent, to avoid this problem of misaligned contained inside the brackets 
```scala
def getAccountSummaries(
  accountIds: List[AccountId],
  identityId: String,
  accountSummaryForAccountId: AccountId => Future[Disjunction[String, AccountSummary]]
)
```